client/src/app/styles/index.scss

```diff
@@ -16,21 +16,19 @@
 @import './utils';
 
 // 3. Containers
-@import '../../modules/legacy/styles/containers/app';
+@import '../../app/components/Layout/Layout.scss';
 
 // 4. Components
-@import '../../modules/legacy/styles/components/AppLogo';
-@import '../../modules/legacy/styles/components/Footer';
-@import '../../modules/legacy/styles/components/Authbox';
-@import '../../modules/legacy/styles/components/googleButton';
-@import '../../modules/legacy/styles/components/Header';
-@import '../../modules/legacy/styles/components/MessageBox';
-@import '../../modules/legacy/styles/components/Preloader';
-@import '../../modules/legacy/styles/components/ProductsContainer';
-@import '../../modules/legacy/styles/components/ProductsList';
-@import '../../modules/legacy/styles/components/InputBar';
-@import '../../modules/legacy/styles/components/UserBar';
-@import '../../modules/legacy/styles/components/SortBox';
-@import '../../modules/legacy/styles/components/VotingBox';
-@import '../../modules/legacy/styles/components/FilterBox';
-@import '../components/Toolbar/Toolbar';
+@import '../../common/components/AppLogo.scss';
+@import '../../app/components/Footer.scss';
```

Comment:
Maybe it's better and cleaner to place in the each module's directory an index.scss file that would contain the import of scss files from a given module. Those module's main scss files could be imported in main index.scss from the app/styles directory.

Reply:
We decided to leave it as it is, casue we want to keep .scss file just next to .jsx file of component...
![zrzut ekranu 2019-01-21 o 13 55 54](https://user-images.githubusercontent.com/27632432/51476080-b620c400-1d84-11e9-9e9d-4d3ae4051dfc.png)